pysindy/feature_library/generalized_library.py

```diff
@@ -232,7 +244,25 @@ def fit(self, x, y=None):
 
         return self
 
-    def transform(self, x):
+    def has_type(self, libtype: Type, exclusively=False) -> bool:
```

Comment:
This is not used yet, but implemented for future purposes? Should the `ConcatLibrary` and `TensorLibrary` also implement it, and should included in `BaseFeatureLibrary`? Also, what if the self.libraries_ contain a `GeneralizedLibrary`, `TensorLibrary`, or `ConcatLibrary`? Should has_type recursively check the `has_type` of `self.libraries_`? It would be easier if we never need to check if the library contains any particular type and we could remove this function...

Reply:
Also a vestige of refactoring, when pysindy.py had several nested loops and conditionals to check whether to treat the data like it went to a `WeakPDELibrary`.

Since there's no current use or test for the function, I'm happy to delete it rather than try to improve it.

Reply:
removed in new commit.